### PR TITLE
Remove "main" : "scripts/w3c-schemas.js"

### PR DESCRIPTION
**Pull request by juanmav**
Remove main:scripts/w3c-schemas.js to prevent bower and wiredep to auto include this file. This file is to load modules in nodejs context but in a browser context fails, because `module` does not exists.

Could we have a new tag with this change? 

Thanks in advance
JM

### CODE REVIEW

**Comment by highsource** [comment]
@juanmav 1.1.1 is released now.

**Comment by juanmav** [comment]
@highsource Thanks!

**Comment by bartvde** [comment]
This will generate a warning but I guess that's no big issue

```
bower w3c-schemas#master                                  invalid-meta w3c-schemas is missing "main" entry in bower.json
```

**Comment by highsource** [comment]
I'm no big bower guru, so I'm not sure how to proceed. We had this #11 and #12 to remove `main`. Should it be removed or should it be kept?

**Comment by bartvde** [comment]
keeping it is worse, I'll investigate a bit to see if there a way to do this without generating a warning.

**Comment by bartvde** [comment]
there doesn't seem to be a way, but in the end it's just a warning, so safe to ignore.

**Comment by highsource** [comment]
Should we a add a dummy index.js instead?

**Comment by juanmav** [comment]
The dummy file can off the warning and be safe in a browser.

**Comment by highsource** [comment]
PR anyone? :)

**Comment by juanmav** [comment]
ok, ok... give me some minutes.

**Comment by highsource** [comment]
Thanks. :)

**Comment by juanmav** [comment]
Maybe will be better to change the w3c-schemas.js and revert to bower.json changes ?

```javascript
if (typeof module !== 'undefined' && module.exports) {
    module.exports.Atom_1_0         = require('./lib/Atom_1_0'        ).Atom_1_0        ;
    module.exports.WS_Addr_1_0_Core = require('./lib/WS_Addr_1_0_Core').WS_Addr_1_0_Core;
    module.exports.XHTML_1_0_Strict = require('./lib/XHTML_1_0_Strict').XHTML_1_0_Strict;
    module.exports.XLink_1_0        = require('./lib/XLink_1_0'       ).XLink_1_0       ;
    module.exports.XSD_1_0          = require('./lib/XSD_1_0'         ).XSD_1_0         ;
} else {
    // Browser context
    // You may include the modules by hand
    console.warn('w3c-schecmas in a browser context, you may include the modules by hand');
}
```

**Comment by highsource** [comment]
Basically an UMD approach... Hm.

**Comment by juanmav** [comment]
@highsource For some reason you reverted the changes? ed36c7ac1744cfcdc3d6241ec3f80f36856c5dfe 

Thanks
JM

**Comment by highsource** [comment]
@juanmav Back then I've finally understood that putting UMD in `w3c-schemas.js` or `ogc-schemas.js` actually does not really make sense. Instead, I've made `main` an array and listed JS files there:

https://github.com/highsource/w3c-schemas/blob/master/bower.json#L8-L14

The reason it does not make sens is that I don't see a scenario when it would work.

In browser, you won't use `w3c-schemas.js` anyway. You'll never use all of the schemas at once, too heavy and absolutely unnecessary. You'll use individual schemas so you don't need this header file.

For bower, you just need a list of JS files in main, the header file itself is not necessary.

For node.js it's okay to have it (you'd still normally use individual schemas, but a single header file is OK). But you won't need UMD then.

So basically I don't see a scenario when UMD is necessary/useful in the header file. This is why I've reverted this and enumerated JS files in `main` array instead. Which solves this issue.
